eesp-ikev2.org

```diff
@@ -253,15 +253,14 @@ alternative to large numbers of bidirectional Child SAs with the
 same traffic selectors [[RFC7296]], [[RFC9611]].
 
 Each Sub SA is identified by a Sub SA ID, which MUST be carried in
-each EESP packet in the Session ID field—consistent with the negotiation of the EESP Child SA.  This
-Sub SA ID is used to derive a unique key, yielding the following
-benefits:
+each EESP packet in the Session ID field—consistent with the
+negotiation of the EESP Child SA.  This Sub SA ID is used to derive
+a unique key, yielding the following benefits:
```

Comment:
Hm, not sure if these benefits result from that unique key. Some are just benefits of having Sub SAs.